core/migrations/0008_instance_actions_and_migrate_provider_trait_to_dns_and_auto_imaging.py

```diff
@@ -100,6 +100,9 @@ def add_instance_actions(Provider, InstanceAction, ProviderInstanceAction):
         name="Reboot",
         description="""Reboots an instance when it is in ANY State""")
     InstanceAction.objects.get_or_create(
+        name="Redeploy",
+        description="""Redeploy to an instance when it is in ANY active state""")
+    InstanceAction.objects.get_or_create(
```

Comment:
This seems problematic, doesn't it? Shouldn't you create a new migration instead of updating an old one? Otherwise it won't run if the database is newer than this, right? Or am I missing something.